mitodl/mitxonline
# Flipping the default for `for_flexible_pricing` from True to False

#### Pre-Flight checklist


#### What are the relevant tickets?

Closes #1265 

#### What's this PR do?

Makes the default for `for_flexible_pricing` in discounts False instead of True.

#### How should this be manually tested?

Try to create a discount code via django admin. It should default to not being for flexible pricing.
